asserts/authority_delegation_test.go

```diff
@@ -88,7 +97,8 @@ func (s *authorityDelegationSuite) TestAuthorityDelegationCheckUntrustedAuthorit
 		"delegate-id": "other",
 		"assertions": []interface{}{
 			map[string]interface{}{
-				"type": "snap-declaration",
+				"type":  "snap-declaration",
+				"since": time.Now().Format(time.RFC3339),
```

Comment:
Nitpick, would be nice to mock timeNow like we do in other places, but I guess it would prompt more updates in asserts, so maybe a followup material. We have enough time until 2032 anyway ;)